src/MyClass.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+    class MyClass {
```

Comment:
Please use more meaningful name

Reply:
ok, i will fix that

Reply:
I believe you forgot to remove this class